examples/polymer-cli/src/stories/addon-links.stories.js

```diff
@@ -0,0 +1,9 @@
+import { storiesOf } from '@storybook/polymer';
+import { linkTo } from '@storybook/addon-links';
+import { document } from 'global';
+
+storiesOf('Addon|Links', module).add('Go to welcome', () => {
+  const el = document.createElement('playground-button');
```

Comment:
Let's use some other element, it's quite weird right now: https://deploy-preview-3096--storybooks-polymer.netlify.com/?selectedKind=Addon%7CLinks&selectedStory=Go%20to%20welcome&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel